math/libfaiss/Portfile

```diff
@@ -47,7 +47,7 @@ configure.args-append \
                     -DFAISS_ENABLE_PYTHON:BOOL=OFF
 
 variant openblas description {Use OpenBLAS instead of Apple's Accelerate framework} {
-    depends_lib-append  port:OpenBLAS
+    depends_lib-append  path:lib/libopenblas.dylib:OpenBLAS
```

Comment:
I *think* you'd need to increase the "revision" for this change to make sure that the dependencies are correctly reflected in the registry.

Reply:
Ok, will rebase today, thank you.

Reply:
@reneeotten Ah, but we update the version here in this PR from 1.7.2 to 1.7.3. Revbump still needed?

Reply:
sorry, I missed that somehow...